### PR TITLE
Fix: Link to issue

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] adds a link to #673

### CODE REVIEW

**Comment by ostrolucky** [comment]
I think it's too late to add reference to 5 year old BC break. Or what's the justification you see for adding this there?

File: ConnectionFactory.php [hunk 1]
```diff
@@ -132,6 +132,8 @@ public function createConnection(array $params, ?Configuration $config = null, ?
      * and the platform version is unknown.
      * For details have a look at DoctrineBundle issue #673.
```
Comment (localheinz) [inline ConnectionFactory.php]: @ostrolucky 

The reference is already here, just not clickable.

Comment (ostrolucky) [reply ConnectionFactory.php]: Hmm I would say perhaps better then to change it to a link rather than adding a link additionally to ID

**Review by ostrolucky — APPROVED** [review]

**Review by dmaicher — APPROVED** [review]

**Comment by localheinz** [comment]
Thank you, @dmaicher and @ostrolucky!

**Pull request merged**